### PR TITLE
design: update about page

### LINKED ISSUE #686: Update About Page
Update the "About Page" regarding the new Layout 
https://www.figma.com/file/FWlI63N28LP7BxqNqj1bGw/sy-004-(Design!)?node-id=6093%3A24595&t=B5lVnr3rV8owVoTF-4

**Changes:**
- No Headline in the Header
- Integrate Google Maps Link in the Intro-Text
- People and Office Images shown in a Collage-Layout

**German Text:**
Das ist unser erfahrenes Team aus Entwicklern und Designern. Einige arbeiten vollständig remote, einige in unserem Büro im Herzen Münchens – direkt am Sendlinger Tor.
Wir freuen uns immer über Besuch und einen gemeinsamen Kaffee.

CleanShot 2022-12-19 at 10 01 29@2x

### PR DESCRIPTION
**Pull request by timonsommer**
The new about page layout using CSS grids for precise placement of the images and a new Contentful media type (`Gallery Image`)

### CODE REVIEW

**Comment by fischeversenker** [comment]
I just had a quick look at the preview deployment and I love it! 😍 🤩

File: src/components/pages/about-us/team.tsx
Comment (georgiee) [inline src/components/pages/about-us/team.tsx]: Interesting. Can you say a few more words about this solution? It's pragmatic, which I love for this case But is this your solution? Did you find it somewhere? How will it overflow with more images? What's the current capacity?

File: src/pages/about-us.tsx
Comment (georgiee) [inline src/pages/about-us.tsx:50]: I see many faces do show a cut off which comes from the design input. Nothing to blame technical-wise. But what about the formats that you have omitted and the placeholder? How did you decide about that?

File: src/pages/about-us.tsx
Comment (georgiee) [inline src/pages/about-us.tsx:36]: Not related to this code line but I have no other place to put this comment inline: 
What's the file `2.png`? It's all black.

**Review by georgiee — COMMENTED** [review]
Can you describe the Contentful Model Changes too? How to author team changes? How to decide about greyscale? Which processor does the greyscale? Sadly, from my perspective, being the photographer, the greyscale looks dull almost bad. I would prefer not to have greyscale at all, but that's not a discussion for here. You did a great job implementing the given design. Coudl you please explain the grid & contentful? After that I'm happy to merge. (hence this is only a review comment without any approval).

**Comment by georgiee** [comment]
One additional question without having access to Contentful right now: Did you change the original images or is some processing happening (i.e. you cropped and did other changes through Contentful?). Thanks for your clarification.

File: src/pages/about-us.tsx
Comment (timonsommer) [reply src/pages/about-us.tsx:36]: I need to remove this file, thank you for the comment. It was used as a static image placeholder to test and visualize the gallery tile layout without having to think about things like cropping and where to get the picture files from.

File: src/components/pages/about-us/team.tsx
Comment (timonsommer) [reply src/components/pages/about-us/team.tsx]: The solution sets up a CSS grid with multiple custom-defined grid areas so that the images can be placed exactly where the design stipulates. The current capacity is 38 rows of atomic tiles. When new images are added to the bottom, any overflowing elements will only be included if there is an area defined for them. Since the mobile version does not use grids, the overflowing image will still be shown on mobile. I realized that this is not optimal and I will try to fix this. Inspired by the CSS grids I used on the BadgeForge project, I came up with this grid-area solution. At the moment it is rather inflexible, so I will look into making editing the grid easier, which could include setting the tile size directly in Contentful.

**Comment by timonsommer** [comment]
> One additional question without having access to Contentful right now: Did you change the original images or is some processing happening (i.e. you cropped and did other changes through Contentful?). Thanks for your clarification.

Currently, all image files are taken directly from Figma. That includes the cropping masks as well as the grayscale effects.

File: src/components/pages/about-us/team.tsx
Comment (timonsommer) [reply src/components/pages/about-us/team.tsx]: As of 66978e4620155aedf04e77378599cf3c9082e863, the tile size can be set directly in Contentful for each image. This removes the need for predefined grid areas, and all overflowing items now will be placed automatically according to their tile size.

**Review by georgiee — CHANGES_REQUESTED** [review]
I would like to hold back this change, mostly related to the given design input rather the technical implementation 🙏

Technically there is one thing: I think the grid and the associated media in contentful (the new `GalleryImage`) is a little bit too pragmatic. I would like to upload the original images and only then apply transformations (crop) or other adjustments (grayscale) either in Contentful or through the processing available in Gatsby. Loosing the original input for images with a CMS that is able to manage then is not acceptable. Let's work on that once the design input got updated. 

Thank you for your help and work you put into this already 👍

**Comment by georgiee** [comment]
I think this branch is obsolete now. @hofmann-felix already took the core parts with the credits due for @timonsommer and put it into another iteration. Thank you for your efforts 🙌

**Pull request closed without merging**
